packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -94,24 +109,27 @@ export default defineConfig({
       await approveWeth()
 
       // Set Cypress variables
-      config.env.ETH_RPC_URL = ethRpcUrl
-      config.env.ARB_RPC_URL = arbRpcUrl
+      config.env.ETH_RPC_URL = isOrbitTest ? arbRpcUrl : ethRpcUrl
```

Comment:
I didn't rename `ETH_RPC_URL` and  `ARB_RPC_URL` for less diff. It can be done as part of bridge renaming PRs